### DISCUSSION TITLE
Sync issue

### DISCUSSION THREAD

**Discussion by Nishant9947** [ROM-builders/temporary#9049] [General]
Bruh i am trying to compile Aosp-Extended A12 (S) but now 3rd time i m getting sync issue.
Can't sync full device tree.
I done everything From my side. 

Check wen h r free thanks😊
Link :- https://cirrus-ci.com/task/5859260264349696?logs=sync#L47

**Comment by Apon77** [comment]
> Bruh i am trying to compile Aosp-Extended A12 (S) but now 3rd time i m getting sync issue. Can't sync full device tree. I done everything From my side.
> 
> Check wen h r free thanks😊 Link :- https://cirrus-ci.com/task/5859260264349696?logs=sync#L47

Please test sync locally first. Issue from your side.

**Reply by Nishant9947** [reply]
Ohkay bruh thanks for replying
